src/components/pages/landing/workshops/WorkshopsList/WorkshopsList.tsx

```diff
@@ -1,45 +1,108 @@
 import { WORKSHOPS_MOCKS } from "@/mocks/workshops"
 import { Link } from "react-router-dom"
+import { useState } from "react";
 import s from "./WorkshopsList.module.css"
 
 const WorkshopsList = () => {
+  const [selectedUniversity, setSelectedUniversity] = useState("");
+  const [selectedField, setSelectedField] = useState("");
+
+  // Pobieranie unikalnych uczelni
+  const universities = [
+    ...new Set(WORKSHOPS_MOCKS.map((workshop) => workshop.university.name)),
```

Comment:
Nie wiem czy jest jest potrzebny bo nigdy nie używałem tej struktury danych w sumie dopytam o to, może Kacper Książek będzie wiedział czy ma sens tego używać.